### PR TITLE
Uniform Update

### PR DESCRIPTION
**Pull request by AMcilraith**
Changes from NTEF Loadout to Fleet Loadout

### CODE REVIEW

File: maps/torch/datums/uniforms_fleet.dm [hunk 1]
```diff
@@ -478,4 +477,76 @@
 
 	dress_over = /obj/item/clothing/suit/storage/solgov/dress/fleet/flag
 	dress_hat = /obj/item/clothing/head/solgov/dress/fleet/command
-	dress_extra = list(/obj/item/weapon/storage/belt/holster/sheath/fleet, /obj/item/clothing/head/beret/solgov/fleet/dress/command)
+	dress_extra = list(/obj/item/weapon/storage/belt/holster/sheath/fleet, ,/obj/item/clothing/head/beret/solgov/fleet/dress/command)
+	
+// have to resprite science, so that'll have to wait
+/decl/hierarchy/mil_uniform/fleet/sci
+	name = "NTEF science"
```
Comment (Anton-Kr) [inline maps/torch/datums/uniforms_fleet.dm]: NTEF name

**Review by Anton-Kr — CHANGES_REQUESTED** [review]

File: maps/torch/datums/uniforms_fleet.dm (hunk 1 above)
Comment (AMcilraith) [reply maps/torch/datums/uniforms_fleet.dm]: This is the default listing, just copied it from the normal use file, and put it here

Dunno what to say tbh

Comment (AMcilraith) [reply maps/torch/datums/uniforms_fleet.dm]: Fixed

File: maps/torch/datums/uniforms.dm [hunk 2]
```diff
@@ -56,9 +56,6 @@
 	dress_gloves = /obj/item/clothing/gloves/white
 
 	dress_extra = list(/obj/item/clothing/accessory/solgov/ec_scarf)
-
-// Commented out because it overrides modular's uniform system in some places.
-/*
```
Comment (ProbablyCarl) [inline maps/torch/datums/uniforms.dm:61]: What's up with uncommenting this?
It's defined elsewhere, in [maps/torch/datums/uniforms_fleet.dm](https://github.com/BoHBranch/BoH-Bay/pull/1484/files#diff-f910937b45d1b5e84d4f5c4fa4b2cb54d2850278f0b387b4ea7818f7b66821b4), and this was commented out for a very good reason.

Comment (AMcilraith) [reply maps/torch/datums/uniforms.dm:61]: Okay, so it's uncommented due to being necessary for the uniform changes. Essentially, this allows me to fix it by re-adding the default fleet uniforms

Comment (AMcilraith) [reply maps/torch/datums/uniforms.dm:61]: Don't worry, cloaks are not gone, just the NTEF uniforms

**Comment by N8-Toe** [comment]
I'm not a fan of the service uniforms with a stripe, I think its un-needed visual noise that lowers the quality of the sprite. 

Otherwise I do appreciate the switchover from the old EC uniforms to fleet in the vendor!

**Comment by AMcilraith** [comment]
Had to rebase due to new updates

**Pull request closed without merging**